tests/chebfun/test_atan2.m

```diff
@@ -8,9 +8,12 @@
 b = 2.25*pi;
 
 %% Scalar-valued, tan(f, g):
-x = chebfun(@(x) x, [a, b], pref);
-f = .5+sin(x).*exp(-.1*x.^2);
-g = cos(x).*(1+x.^2);
+%x = chebfun(@(x) x, [a, b], pref);
+%f = .5+sin(x).*exp(-.1*x.^2);
+f = chebfun('.5+sin(x).*exp(-.1*x.^2)', [a, b]);
+
+%g = cos(x).*(1+x.^2);
+g = chebfun('cos(x).*(1+x.^2)', [a, b]);
 h = atan2(f, g);
```

Comment:
@kolmanthomas what is this about?  I gather there is some error about complex values when I try this?

Reply:
I played with this and filed #8 based on what I found so far.